src/App.tsx

```diff
@@ -1,14 +1,8 @@
-import { useTranslation } from 'react-i18next'
-
+import { Sidebar } from '@components'
 export const App = () => {
-  const { t, i18n } = useTranslation()
   return (
```

Comment:
```suggestion
  return <Sidebar />
```
Nếu chỉ có 1 children thì không cần bọc Fragment

Reply:
fixed